fenix/app/src/androidTest/java/org/mozilla/fenix/ui/NavigationToolbarTest.kt

```diff
@@ -258,4 +263,102 @@ class NavigationToolbarTest {
             verifyClearSiteDataPrompt(helpPageUrl)
         }
     }
+
+    @SmokeTest
+    @Test
+    fun verifyNavigationToolbarButtonsTest() {
+        val defaultWebPage = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(defaultWebPage.url) {
+            verifyNavigationToolbarButtons()
+        }
+    }
+
+    @SmokeTest
+    @Test
+    fun goToHomeScreenTest() {
+        val genericURL = TestAssetHelper.getGenericAsset(mockWebServer, 1)
+
+        navigationToolbar {
+        }.enterURLAndEnterToBrowser(genericURL.url) {
+            mDevice.waitForIdle()
+        }.goToHomescreen {
+            verifyHomeScreen()
+        }.openThreeDotMenu {
+        }.openSettings {
+        }.openCustomizeSubMenu {
+            clickTopToolbarToggle()
```

Comment:
I would remove the top toolbar check in these tests, as I don't think it is too relevant. We have other tests for the top toolbar position.

Reply:
As discussed, I've removed both the step from the manual and removed the corresponding automated test.